docs/content/dagster-plus/account/authentication/okta/saml-sso.mdx

```diff
@@ -78,20 +63,23 @@ To complete the steps in this guide, you'll need:
 
 Next, you'll save and upload the application's SAML metadata to Dagster+. This will enable single sign-on.
 
-1. In Okta, navigate to the Dagster+ application.
+1. In the **Sign On Settings**, navigate to the **SAML Signing Certificates** section.
+
+2. Click the **Actions** button of the **Active** certificate.
+
+3. Click **View IdP metadata**:
+
+   ![The View IdP metadata options in the Okta UI](/images/dagster-cloud/sso/okta/save_identity_provider_metadata.png)
 
-2. Navigate to **Sign On**.
+   This will open a new page in your browser with the IdP metadata in XML format.
 
-3. Click **Identity Provider metadata** to initiate a download. This will save the SAML metadata file to your computer.
+4. Right-click on the page and use **Save As** or **Save Page As**:
 
-   <Image
-   alt="Okta Save Identity Provider Metadata"
-   src="/images/dagster-cloud/sso/okta/save_identity_provider_metadata.jpg"
-   width={1431}
-   height={687}
-   />
+   ![](/images/dagster-cloud/sso/okta/save-xml-as-page.png)
 
-4. After you've downloaded the SAML metadata file, upload it to Dagster+ using the `dagster-cloud` CLI:
+   In Chrome and Edge, the file will be downloaded as an XML file. In Firefox, choose **Save Page As > Save as type**, then select **All files**. **Note**: Copying and pasting the metadata can cause formatting issues that will prevent successful setup. Saving the page directly from the browser will avoid this.
+
+5. After you've downloaded the metadata file, upload it to Dagster+ using the `dagster-cloud` CLI:
```

Comment:
i really wish this could be done in the UI

Reply:
Me too. I have no idea why this is 'better' :|